extensions/smallrye-reactive-messaging/runtime/src/main/java/io/quarkus/smallrye/reactivemessaging/runtime/QuarkusWorkerPoolRegistry.java

```diff
@@ -20,18 +16,15 @@
 import jakarta.inject.Inject;
 
 import org.eclipse.microprofile.config.ConfigProvider;
-import org.jboss.logging.Logger;
 import org.slf4j.LoggerFactory;
```

Comment:
Why do we depends on SLF4J?

Reply:
I haven't noticed that it was used. Will change it to jboss logger.